### PR TITLE
Match frontend

### PR DESCRIPTION
**Pull request by thijnmens**
I also ran the rider codeformatter over the project because the indentation was all over the place, so files with only formatting changes you can just ignore and accept, this is the only pull request that will have these

### CODE REVIEW

File: Controllers/WeatherForecastController.cs
Comment (thijnmens) [inline Controllers/WeatherForecastController.cs:1]: Formatting only

File: OpenRemoteAPI.csproj
Comment (thijnmens) [inline OpenRemoteAPI.csproj:1]: Formatting only

File: Properties/launchSettings.json
Comment (thijnmens) [inline Properties/launchSettings.json:1]: Formatting only

File: WeatherForecast.cs
Comment (thijnmens) [inline WeatherForecast.cs:1]: Formatting only

File: appsettings.Development.json
Comment (thijnmens) [inline appsettings.Development.json:1]: Formatting only

File: appsettings.json
Comment (thijnmens) [inline appsettings.json:1]: Formatting only

**Review by thijnmens — COMMENTED** [review]
I've marked all the files where the changes are only formatting changes, Just resolve these comments when you review 👍

**Review by Tais993 — CHANGES_REQUESTED** [review]
Can you rebase it, so there's 3 separate commits.

1 for the gitignore
1 for the formatting
1 for the changes you made

The files you changed and formatted can just be added to the third commit, don't need to bother all that out. Then we can rebase this branch onto develop.

The changes itself are fine

**Comment by thijnmens** [comment]
Had trouble editing the past commits so i redid the changes in another branch, #4

**Pull request closed without merging**